BLOGPOST.md

```diff
@@ -0,0 +1,38 @@
+# Introducing the SAP Cloud SDK for AI (JavaScript/TypeScript) 🎉
+
+We are excited to announce the initial release of the [SAP Cloud SDK for AI](https://github.com/SAP/ai-sdk-js#readme)! It simplifies the integration of generative AI capabilities into your SAP Business Technology Platform (BTP) applications.
+
+The SDK allows you to leverage the SAP AI Core and SAP AI Launchpad, enabling capabilities like templating, grounding, data masking, and more. In this blog post, we’ll introduce you to the key packages and their features.
+
+## Key Features
+### AI Orchestration with `@sap-ai-sdk/orchestration`
+The [@sap-ai-sdk/orchestration](https://github.com/SAP/ai-sdk-js/tree/main/packages/orchestration#readme) package enables generative AI orchestration, allowing you to configure templating, content filtering, and data masking for your applications. With its orchestration service, you can streamline workflows and manage AI models within SAP AI Core.
+
+- **Templating**: Create dynamic prompts with placeholders to customize AI interactions based on user inputs.
+- **Content Filtering**: Apply filters to ensure input and output adhere to content safety guidelines.
+- **Data Masking**: Anonymize sensitive information with data masking features.
+
+### AI Management with `@sap-ai-sdk/ai-api`
+The [@sap-ai-sdk/ai-api](https://github.com/SAP/ai-sdk-js/tree/main/packages/ai-api#readme) package provides comprehensive tools for managing scenarios and workflows in SAP AI Core. You can automate processes such as creating artifacts, configurations, and deployments; executing batch inference jobs; and managing Docker registries and object storage for training data.
+
+- **Artifact Management**: Register and manage datasets and other model artifacts.
+- **Configuration Management**: Set up configurations for different models and use cases.
+- **Deployment Management**: Deploy AI models and manage their lifecycle within SAP AI Core.
```

Comment:
[pp] Consider linking to readme of the ai core client for specific code samples related to each section.